socorro/external/fs/crashstorage.py

```diff
@@ -663,68 +659,115 @@ class TarFileCrashStore(CrashStorageBase):
         default='gzip',
         from_string_converter=class_converter
     )
-    required_config.add_option(
-        name='os_module',
-        doc='a module that supplies the os interface',
-        default='os',
-        from_string_converter=class_converter
-    )
 
+    #------------------------------------------------------------------------------
     @staticmethod
     def stringify_datetimes(obj):
         if isinstance(obj, datetime.datetime):
-            return obj.strftime("%Y-%m-%d %H:%M:%S.%f")
-        raise TypeError
+            return obj.strftime("%Y-%m-%dT%H:%M:%S.%f")
```

Comment:
that is an error.  I believe the second is supposed to raise a ```TypeError``` rather than defer to the base class.  I'll have to reread the docs on tarfiles.